src/EnergyPlus/DataSurfaces.hh

```diff
@@ -1577,6 +1577,23 @@ namespace DataSurfaces {
         }
     };
 
+    struct IntMassObject
+    {
+        // Members
+        std::string Name;
+        std::string ZoneOrZoneListName; // zone or zone list name
+        int ZoneOrZoneListPtr;          // pointer to a zone list
+        int NumOfZones;                 // number of zones in a zone list
+        int Construction;               // pointer to contruction object
+        Real64 GrossArea;               // internal surface area, [m2]
+        bool ZoneListActive;            // flag to a list
+
+        // Default Constructor
+        IntMassObject() : ZoneOrZoneListPtr(0), NumOfZones(0), Construction(0), GrossArea(0.0), ZoneListActive(false)
+        {
+        }
+    };
```

Comment:
This isn't creating a new IDD object right?  Just allowing a zone list.  I'm surprised to see an entirely new struct being added.  What was the reason for not just altering the existing struct?

Reply:
No it is not creating a new object. But we did not have any struct for InternalMass object previously. The parameters read were passed  to Surface objects on the fly. Now we have to read InternalMass objects and save the data as the first step, and parse the zone list and pass the info in the Surface data object as the second step. It is for this reason that I needed to create new struct.